### PR TITLE
add kscript

### PR DESCRIPTION
**Pull request by fabriziofortino**

### CODE REVIEW

File: plugins/kscript [hunk 1]
```diff
@@ -0,0 +1 @@
+repository = https://github.com/edgelevel/asdf-kscript
```
Comment (Stratus3D) [inline plugins/kscript]: This should be the URL to the Git repository itself, which for Github project would be `https://github.com/edgelevel/asdf-kscript.git`.

**Comment by Stratus3D** [comment]
In the `.travis.yml` file I see a line that references another repo: `asdf plugin-add kotlin https://github.com/missingcharacter/asdf-kotlin.git`. Can this file be deleted or can this line be corrected?

**Comment by Stratus3D** [comment]
Other than those two things the plugin looks good!

File: plugins/kscript (hunk 1 above)
Comment (fabriziofortino) [reply plugins/kscript]: Done

**Comment by fabriziofortino** [comment]
@Stratus3D thanks for reviewing this. Re the `.travis.yml` file, `kscript` depends on `kotlin`. It would not work without it.

**Comment by Stratus3D** [comment]
@fabriziofortino my mistake. Mis-read the `.travis.yml` file. Thanks for the PR!

**Pull request merged**